docs/spec/protocol.rst

```diff
@@ -266,25 +266,25 @@ Generic protocols are important. For example, ``SupportsAbs``, ``Iterable``
 and ``Iterator`` are generic protocols. They are defined similar to normal
 non-protocol generic types::
 
-  class Iterable(Protocol[T]):
+  class Iterable[T](Protocol):
       @abstractmethod
       def __iter__(self) -> Iterator[T]:
           ...
 
-``Protocol[T, S, ...]`` is allowed as a shorthand for
-``Protocol, Generic[T, S, ...]``. It is an error to combine
-``Protocol[T, S, ...]`` with ``Generic[T, S, ...]``, or with the new syntax for
-generic classes in Python 3.12 and above::
+The older syntax ``Protocol[T, S, ...]`` remains available as a shorthand for
+``Protocol, Generic[T, S, ...]``. It is an error to combine the shorthand with
+``Generic[T, S, ...]`` or to mix it with the new ``class Iterable[T]`` form::
 
-  class Iterable(Protocol[T], Generic[T]):   # INVALID
+  class Iterable[T](Protocol, Generic[T]):   # INVALID
```

Comment:
I agree that this modified example demonstrates something a type checker should reject, but I don't think it demonstrates the same thing that it used to (and it seems to me that it makes the example immediately below it redundant). I'd revert this change